lib/internal/Magento/Framework/Mview/View/Collection.php

```diff
@@ -96,7 +96,7 @@ private function getOrderedViewIds()
             $orderedViewIds[] = $indexer->load($indexerId)->getViewId();
         }
         $orderedViewIds = array_filter($orderedViewIds);
-        $orderedViewIds += array_diff(array_keys($this->config->getViews()), $orderedViewIds);
```

Comment:
Hi @Maddesto. Thanks for the collaboration. Can you please explain your issue and fix, as you write in description 
> if $orderedViewIds and result of array_diff has same keys,

but looking to code we get diffs with orderedViewIds  `array_diff(array_keys($this->config->getViews()), $orderedViewIds)`, so array $orderedViewIds and diff with $orderedViewIds can't have the same keys